tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Method_Test.php

```diff
@@ -165,6 +167,21 @@ class WC_Stripe_UPE_Payment_Method_Test extends WC_Mock_Stripe_API_Unit_Test_Cas
 		],
 	];
 
+	/**
+	 * Base template for Stripe's Klarna
+	 */
+	const MOCK_KLARNA_PAYMENT_METHOD_TEMPLATE = [
+		'id'                                  => 'pm_mock_payment_method_id',
+		'type'                                => WC_Stripe_Payment_Methods::KLARNA,
+		WC_Stripe_Payment_Methods::KLARNA => [
+			'dob' => [
+				'day'   => 1,
+				'month' => 1,
```

Comment:
Thought: it may make sense to set this to 2 or some value other than 1, just to make sure we never mix up the day and month.
```suggestion
				'month' => 2,
```